ctools/defragment_sharded_collection.py

```diff
@@ -504,6 +496,7 @@ def lookahead(iterable):
                 consecutive_chunks = [c]
                 estimated_size_of_consecutive_chunks = args.phase_1_estimated_chunk_size_kb
                 if not has_more:
+                    await update_chunk_size_estimation(c)
```

Comment:
Here we were not writing the chunk size estimation for the last chunk, not really a bug, but subsequent loads of `config.chunks` would find a chunk without size estimation

Reply:
ah was  this causing the non-idempotent behaviour?

Reply:
No this is not the main problem, I'll send a subsequent PR with the actual fix for the idempotency ;)